cmd/cmdcommon/cmdcommon.go

```diff
@@ -166,3 +166,11 @@ func updateFlagSet(flags ...*flag.FlagSet) {
 		pflag.CommandLine.AddGoFlagSet(f)
 	}
 }
+
+func getOperatorNamespaceFromEnv() (string, error) {
+	if namespace, ok := os.LookupEnv(hcoutil.OperatorNamespaceEnv); ok {
+		return namespace, nil
+	}
+
+	return "", fmt.Errorf("%s unset or empty in environment", hcoutil.OperatorNamespaceEnv)
```

Comment:
if `hcoutil.OperatorNamespaceEnv` is set but empty, it will return an empty string without an error.

Reply:
Anyway, this is checked on boot (cmd/*/main.go). The boot fails if the env var is not set.

Reply:
I can change that. But this is the same as what was don in the pkg/util before this change, so there is no behavior change here. it's exactly the same logic.

Reply:
@orenc1 - done